integration_tests/test_contract.py

```
@@ -150,44 +158,50 @@ async def test_flow(mantra, connect_mantra):
                 data=weth.fns.transferFrom(
-                    MULTICALL3_ADDRESS, users[0], amount_all
+                    MULTICALL3_ADDRESS, users[0].address, amount_all
                 ).data,
             ),
         ]
     ).transact(w3, users[0])
     await weth.fns.withdraw(amount_all).transact(w3, users[0], to=WETH_ADDRESS)
-    assert all(x == 0 for x in await multicall(w3, balances))
-    assert await balance_of(w3, WETH_ADDRESS, MULTICALL3_ADDRESS) == 0
+    balances_bf = await multicall(w3, balances)
+    await balance_of(w3, WETH_ADDRESS, MULTICALL3_ADDRESS) == 0
```

Ruff (0.12.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)